### DISCUSSION TITLE
UX: Snapshots vs Commits

### DISCUSSION THREAD

**Discussion by SylonZero** [Kong/insomnia#4931] [Feedback]
The UI should really use a standard term like **Commit** instead of **Snapshot**. Everything else is identical to terminology in Git that most devs are familiar with - branches, master, push, pull, etc. It would have been obvious to developers how to use this feature if the word _snapshot_ hadn't been used.

**Comment by filfreire** [comment]
Heads up to @wdawson and the team @Kong/team-insomnia,I've closed the original ticket on this (https://github.com/Kong/insomnia/issues/4026) and switching this to a discussion instead.

We may want to consider updating the terminology we use on Insomnia Sync feature as @SylonZero suggests.
